### ISSUE TITLE
parallel VP runs and gas accounting

### ISSUE THREAD

**Issue by tzemanovic** [anoma/anoma#293]
We might want to set e.g. maximum parallelism to some reasonable value that would be part of the minimum recommended spec for validator hardware. 

The gas accounting formulae for parallelized VPs runs should then be adjusted, so that the parallelized runs are discounted only for the number of runs up to this limit (we currently take the full gas from the VP that used the most gas and divide the rest by a constant).

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201757896515792/1201762884734255) by [Unito](https://www.unito.io)

**Comment by cwgoes** [comment]
Yes; this can be a configurable parameter but we should ensure that the gas costs reflect the actual underlying execution speed.
